assets/css/filtersection.css

```diff
@@ -55,7 +38,353 @@
     background-color: var(--verylightgray);
 }
 
-/* Show the dropdown menu (use JS to add this class to the .dropdown-content container when the user clicks on the dropdown button) */
-.show {
+#filters .filter-links {
+    position: relative;
+    margin-bottom: 30px; 
+}
+
+.filter-links.size-toggle {
+    position: relative;
+    vertical-align: top;
+    padding: 0;
+}
+
+.filter-links.size-toggle a {
+    position: relative;
+}
+
+.filter-links a span {
+    font-weight: 500; 
+    
+}
+
+#filters {
+    width: 16.25%;
+    max-width: 180px;
+    /* height: calc(100vh - 110px); */
+    min-height: 560px;
+    float: left;
+    margin: 0;
+    clear: right;
+    -moz-transform: translateY(0);
```

Comment:
If you have a build process, like webpack, you can use an auto-prefixer to avoid all these browser specific properties.

In this specific case, I think you can just use `transform` without the alternatives.